HW0/P4/P4.py

```diff
@@ -1,7 +1,25 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
+import numpy.linalg as la
 
```

Comment:
I would just add a comment or a docstring to each function describing what it does.